content/en/nodes.md

```diff
@@ -22,5 +21,4 @@ Thus, nodes run apps or processes as part of a group forming a [cluster](/cluste
 
 A node gives you a distinct unit of compute (memory, CPU, network) that you can assign to a cluster. 
 In a [cloud native](/cloud-native-tech/) platform or app a node represents a single unit that can perform work. 
-Ideally, individual nodes are undifferentiated in that 
-any one node of a particular type is indistinguishable from any other node of the same type.
+Ideally, individual nodes are undifferentiated in that any one node of a particular type is indistinguishable from any other node of the same type.
```

Comment:
For reviewers: no content update made here, just a semantic line break adjustment.